src/pages/account/findPassword.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+import { FormProvider, useForm } from 'react-hook-form';
+import type { NextPage } from 'next/types';
+import type { RegisterForm } from 'types/register';
+import { FindPasswordForm } from 'components/account';
+import { Button, Seo } from 'components/common';
+import { Header, HeaderLeft, HeaderTitle } from 'components/layouts';
+import { Z_INDEX } from 'constants/styles';
+
+const FindPassword: NextPage = () => {
+  /**
+   * @todo
+   * RegisterForm 대신 PasswordFindForm 정의하여 사용
+   */
+  const methods = useForm<RegisterForm>({ mode: 'onChange' });
+  const {
+    formState: { isValid },
+  } = methods;
+
+  return (
+    <>
+      <Seo title={'비밀번호 찾기 | a daily diary'} />
+      <Header
+        left={<HeaderLeft type="이전" />}
+        title={<HeaderTitle title={'비밀번호 찾기'} position={'left'} />}
+      />
+      <ContentWrapper>
+        <Title>비밀번호를 잊으셨나요?</Title>
+        <DescriptionText>
+          {`이메일 주소를 입력하면 암호를 재설정할 수 있는 
+            링크를 이메일로 보내드릴게요.`}
```

Comment:
p3;
개행 문자 넣어주시거나 `br` 태그 사용하면 좋을 것 같습니다!

```suggestion
          {`이메일 주소를 입력하면 암호를 재설정할 수 있는\n링크를 이메일로 보내드릴게요.`}
```

Reply:
저는 개행이 있으면 코드에서 보여지는 그대로 화면에도 보여지는 것을 선호해서 `white-space: pre-line` 속성을 썼었는데
한 줄 정도는 `br` 태그 사용으로 변경해도 지저분해 보이지 않는 것 같아 변경하였습니다!

Reply:
저의 경우 최근에는 `\n`나 `br` 태그로 사용하는 것을 선호하게 되었어요!
불필요한 코드를 최소화하는게 가장 좋은 성능 최적화라고 생각해서 가독성을 해치지 않는 선에서 최소화 하려고 하다보니 선호하는 방식이 바뀐 것 같아요.
소현님 의견 남겨주셔서 감사합니다!